### PR TITLE
Adding desktop first media queries and styles

### PR DESCRIPTION
**Pull request by pete-murphy**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1,3 +1,399 @@
...
+}
+
+.top-nav {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  width: var(--main-width);
+}
+
+@media (max-width: 800px) {
```
Comment (ch-ance) [inline css/index.css:126]: Nice job putting these media queries just below their desktop styling. You can also put the media queries at the very bottom of your CSS, but doing it the way you have here is generally easier to read.

File: css/index.css [hunk 2]
```diff
@@ -1,3 +1,399 @@
...
+  .top-nav nav {
+    width: 100%;
+    justify-content: space-between;
+  }
+}
+
+@media (max-width: 500px) {
+  .top-nav nav {
+    flex-direction: column;
```
Comment (ch-ance) [inline css/index.css:148]: 🔥

File: css/index.css [hunk 3]
```diff
@@ -1,3 +1,399 @@
...
+}
+
+@media (max-width: 500px) {
+  .top-nav nav a {
+    text-align: center;
+    padding: 1.26rem 0;
+    border-top: 1px solid var(--gray);
+    border-bottom: 1px solid var(--gray);
+  }
+  .top-nav nav a+a {
```
Comment (ch-ance) [inline css/index.css:183]: I don't see the purpose of `+a` here
```suggestion
  .top-nav nav a { 
```

File: css/index.css [hunk 4]
```diff
@@ -1,3 +1,399 @@
...
+  font-size: 4.5rem;
+}
+
+button {
+  background: white;
+  margin-top: 1rem;
+}
+
+button:hover {
+  background: var(--main-color);
```
Comment (ch-ance) [inline css/index.css:229]: CSS variables, very cool!

File: mobile-first/product.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (ch-ance) [inline mobile-first/product.html:6]: Viewport meta tag, yay!

File: mobile-first/index.css [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+  background: #fffffffa;
+  display: flex;
+  height: var(--top-nav-height);
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+@media (max-width: 500px) {
```
Comment (ch-ance) [inline mobile-first/index.css:112]: mobile first should use min-width media queries!

**Review by ch-ance — APPROVED** [review]
Really great job Peter. I had to Google a couple things in your CSS because I'd never seen them before! Your site is super responsive and looks great at all sizes. Your mobile first directory doesn't appear to be working currently, I would look into using min-width media queries instead of max-width.

**Comment by pete-murphy** [comment]
> Really great job Peter. I had to Google a couple things in your CSS because I'd never seen them before! Your site is super responsive and looks great at all sizes. Your mobile first directory doesn't appear to be working currently, I would look into using min-width media queries instead of max-width.

Yep! I just copied the files over before editing them, haven't made any changes to make it mobile-first yet. Thanks for the feedback!

File: css/index.css (hunk 3 above)
Comment (pete-murphy) [reply css/index.css:183]: Just trying out that query selector, it only selects those that have an `a` directly before it, so essentially all but the first-of-type. I agree it’s not really necessary, but I was just experimenting to see how it worked 🙃 (also so I remember how to use it when I need it)

**Pull request merged**